requirements/dev.in

```diff
@@ -10,7 +10,7 @@ ipdb
 isort
 jsonschema~=2.5
 pep8
-pip==20.2.4
+pip==20.3.4
```

Comment:
pip needs to be updated in the Dockerfile as well, but we also need to fix https://github.com/mozilla/sumo-project/issues/752 to do that.

I'd stick to bumping lxml individually for now with `pip-compile-multi --no-upgrade -g default`